service/ziti-tunnel/service/ipc.go

```diff
@@ -90,6 +90,7 @@ func SubMain(ops chan string, changes chan<- svc.Status, winEvents <-chan Window
 	// initialize the network interface
 	err := initialize(cLogLevel)
 	if err != nil {
+		rts.Close()
```

Comment:
maybe it would make more sense to just `defer rts.Close()` ?

Reply:
probably just before initialize is called. when this function exits - the service is shutting down. seems like it makes sense to defer it

Reply:
done